core/src/main/scala/org/apache/spark/Aggregator.scala

```diff
@@ -34,10 +34,6 @@ case class Aggregator[K, V, C] (
     mergeValue: (C, V) => C,
     mergeCombiners: (C, C) => C) {
 
-  @deprecated("use combineValuesByKey with TaskContext argument", "0.9.0")
-  def combineValuesByKey(iter: Iterator[_ <: Product2[K, V]]): Iterator[(K, C)] =
-    combineValuesByKey(iter, null)
```

Comment:
Ha, the old code here doesn't even make sense: why not pass `TaskContext.get()` instead of null? Glad to see this gone.